knowledge/craftable_spawneggs.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: craftable_spawners
+parent: Knowledge
```

Comment:
I think this should go in the tutorials section. I will be re-organizing stuff soon, but for now I'm trying to keep stuff like this in tutorials. 

Knowledge is for "what is this thing"
Tutorials is for "how do I do this thing"